<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraphicClass</class>
 <widget class="QMainWindow" name="GraphicClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>389</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_2">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item row="0" column="3">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="2">
           <widget class="QComboBox" name="statusComboBox">
            <property name="enabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_4">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Icon</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="statusLineEdit">
            <property name="text">
             <string>Status here</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="userNameLabel">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>User</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="0">
           <widget class="QTabWidget" name="FriendTabWidget">
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="Friends">
             <attribute name="title">
              <string>Friends</string>
             </attribute>
             <layout class="QGridLayout" name="gridLayout_9">
              <item row="0" column="0" colspan="2">
               <widget class="FriendListWidget" name="friendListWidget"/>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="Nothing">
             <attribute name="title">
              <string>Nothing</string>
             </attribute>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
        <item row="3" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_2"/>
        </item>
        <item row="1" column="0">
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="2" column="0" colspan="5">
           <widget class="Line" name="line_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="callFriendLeftPushButton">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Call</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="addFriendPushButton">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Add</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QPushButton" name="deleteFriendPushButton">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Delete</string>
            </property>
           </widget>
          </item>
          <item row="1" column="4">
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="0">
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="0" colspan="5">
           <widget class="Line" name="line_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="1" rowspan="4">
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="selectedFriendNameLabel">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="autoFillBackground">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Selected Friend</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="selectedFriendPersonalMsgLabel">
              <property name="font">
               <font>
                <family>MS PGothic</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Selected Friend Personal Message</string>
              </property>
             </widget>
            </item>
            <item row="0" column="4">
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="2" column="0" colspan="5">
             <widget class="Line" name="line_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QPushButton" name="hangUpPushButton">
              <property name="text">
               <string>Hang Up</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QPushButton" name="callFriendRightPushButton">
              <property name="text">
               <string>Call</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="selectedFriendIconStatusLabel">
              <property name="font">
               <font>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Icon</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QTextBrowser" name="friendMsgBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>2</horstretch>
              <verstretch>3</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLineEdit" name="sendBoxTextEdit"/>
          </item>
          <item>
           <widget class="QPushButton" name="sendBoxPushButton">
            <property name="text">
             <string>Send</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>670</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuVBabel">
    <property name="title">
     <string>VBabel</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuConnection">
    <property name="title">
     <string>Connection</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionLogin"/>
    <addaction name="actionLogout"/>
   </widget>
   <widget class="QMenu" name="menuManage_account">
    <property name="title">
     <string>Manage account</string>
    </property>
    <addaction name="actionCreateAccount"/>
    <addaction name="actionDeleteAccount"/>
    <addaction name="actionAccountOptions"/>
   </widget>
   <widget class="QMenu" name="menuVoicemail">
    <property name="title">
     <string>Voicemail</string>
    </property>
    <addaction name="actionSetAnswer"/>
    <addaction name="actionUnSetAnswer"/>
   </widget>
   <widget class="QMenu" name="menuFriends_list">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>Friends list</string>
    </property>
    <addaction name="actionAddFriend"/>
   </widget>
   <addaction name="menuVBabel"/>
   <addaction name="menuConnection"/>
   <addaction name="menuManage_account"/>
   <addaction name="menuFriends_list"/>
   <addaction name="menuVoicemail"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionConnect">
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionCreateAccount">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Create account</string>
   </property>
  </action>
  <action name="actionLogin">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Log in</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSetAnswer">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Set voicemail</string>
   </property>
  </action>
  <action name="actionUnSetAnswer">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Unset voicemail</string>
   </property>
  </action>
  <action name="actionCall">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Call</string>
   </property>
  </action>
  <action name="actionDeleteAccount">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Delete account</string>
   </property>
  </action>
  <action name="actionAddFriend">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Add friend</string>
   </property>
  </action>
  <action name="actionDelete_friend">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Delete friend</string>
   </property>
  </action>
  <action name="actionLogout">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Log out</string>
   </property>
  </action>
  <action name="actionAccountOptions">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Account options</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>FriendListWidget</class>
   <extends>QListWidget</extends>
   <header>FriendListWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="MainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
